lib/rubocop/cop/performance/caller.rb

```diff
@@ -44,15 +48,16 @@ def on_send(node)
         private
 
         def message(node)
+          method_name = node.receiver.method_name
           caller_arg = node.receiver.arguments[0]
```

Comment:
There's a `#first_argument` method on parameterized nodes, that could be used here, and on L:56. 😊